inst/pages/Introductory-workflow-french-version.qmd

```diff
@@ -0,0 +1,333 @@
+## Introductory (French)
+
+```{r setup, echo=FALSE, results="asis"}
+library(rebook)
+chapterPreamble()
+```
```

Comment:
This is now in Rmarkdown-style

Can you  put it in quarto-style?

Can you name this chunk

#| label: setup
#| echo: FALSE
#| results: "asis"

might work